sandman_web/sandman_web/templates/base.html

```diff
@@ -8,6 +8,9 @@ <h1>Sandman</h1>
   <ul>
       <li><a href="{{ url_for('reports.index') }}">Reports</a>
   </ul>
+  <ul>
+    <li><a href="/rhasspy" target="blank">Rhasspy</a></li>
```

Comment:
Aren't you supposed to use the url_for to target routes?

Reply:
Functionally both ways work. Since a route was created for this to work using url_for probably is the correct way of implementing this.